big_tests/tests/mongoose_helper.erl

```diff
@@ -351,16 +351,15 @@ get_session_pid(User, Node) ->
     Resource = escalus_client:resource(User),
     Username = escalus_client:username(User),
     Server = escalus_client:server(User),
-    successful_rpc(Node, ejabberd_sm, get_session_pid,
```

Comment:
Can we create the JID on the test node instead of calling MongooseIM? We are trying to reduce number of RPC calls in our tests.

Reply:
For some strange reason, I tried just manually doing `{jid, U, S, R, U, S, R}`, which is pretty much what MIM expects, and I had strange errors. Another way would be adding such functionality to escalus, to make real jids, but that was perhaps a bit too much for this already huge PR. I thought of just having the rpc call for now, until we do this in escalus later on.

Reply:
Using `{jid, U, S, R, U, S, R}` will not work for users like "Alice@locacalHOst". In the simplest case the jid would look like this:
`{jid, <<"Alice">>, <<"localHOst">>, <<>>, <<"alice">>, <<"localhost">>, <<>>}`

There is simplistic function which lowers strings `escalus_utis:jid_to_lower`.

Reply:
Given big tests now use rebar3 reliance on stringprep is not a big deal anymore, it can be just another dependency. The problem, though, is that part of the XML/XMPP/JID handling code in the server and in the client are separate, e.g. the `jid` module.

Reply:
I'd rather we copied over the `jid` module to big tests and rename appropriately, e.g. to `mim_big_tests_jid` and just call directly, instead of introducing one more RPC call to each already existing `make_session`/`get_session`.

Later, moving `jid` to `exml`, or to `escalus`, or a possibly new `xmpp` lib will be easier than rewriting all these new RPC-make-jid calls.

Reply:
At the very least, we could have the `rpc(...)` call hidden behind `mongoose_helper:make_jid(Node, [User, Server, Resource])` so it's easier to maintain&refactor later.